internal/config/toml_test.go

```diff
@@ -74,6 +74,9 @@ spec-dirs = ["/etc/cdi", "/var/run/cdi"]
 [nvidia-container-runtime.modes.csv]
 mount-spec-path = "/etc/nvidia-container-runtime/host-files-for-container.d"
 
+[nvidia-container-runtime.modes.jit-cdi]
+load-kernel-modules = ["nvidia", "nvidia-uvm", "nvidia-modeset"]
```

Comment:
How did we end up making this fine selection? :wine_glass: :grapes: :cheese:

Reply:
These are the kernel modules that are required for various GPU functionalities. In the `nvidia-container-cli` we do this through `nvidia-modprobe`:

* `nvidia`: https://github.com/NVIDIA/libnvidia-container/blob/95d3e86522976061e856724867ebcaf75c4e9b60/src/nvc.c#L279
* `nvidia-uvm`: https://github.com/NVIDIA/libnvidia-container/blob/95d3e86522976061e856724867ebcaf75c4e9b60/src/nvc.c#L305
* `nvidia-modeset`: https://github.com/NVIDIA/libnvidia-container/blob/95d3e86522976061e856724867ebcaf75c4e9b60/src/nvc.c#L314

Reply:
This is actually a typo.